tests/test_blockchain_utils.py

```diff
@@ -2,9 +2,13 @@
 import ipfsapi
 import pytest
 
-from core.blockchain.blockchain_utils import *
+from core.blockchain.blockchain_utils import getter, setter
 from core.configuration import ConfigurationManager
 
+test_nonexistent_key = 'nonexistence'
+test_single_key = 'singleton'
+test_multiple_key = 'multiplicity'
+test_value = 'World!'
```

Comment:
I considered this but they're more useful for constants reused across multiple test files, and also add overheard relative to just adding constants at the top of the file.

Reply:
overall, I'm not satisfied with this and the other comment on how we're dealing with constants in tests. Some changes I propose:
- capitalize constant values so at least they conform to how we do constants elsewhere
- give these constants more generic names (e.g. TEST_VAL_A) so that they can be differentiated within a test but used across different tests

Will ask for @georgymh's opinion as I know he brought this change up in the first place

Reply:
Done